src/components/headerImage.tsx

```diff
@@ -74,7 +74,7 @@ const imgStyles = (width: number, height: number): SerializedStyles => css`
 
 const immersiveImgStyles = css`
     display: block;
-    height: 100vh;
+    height: 80vh;
```

Comment:
Would you be able to bump this up very slightly? @blongden73 told me the idea is the bottom of the page should be flush with the baseline of the second line of text, i.e. the top of the nav should hit (but not cover) approximately the bottom of the "m" in the word "masked" in your screenshot.

Reply:
Is this another app vs dotcom style question. The rule for the current templates was to show the first two lines of the headline...

Sorry for another tag @benwuersching 😀

Reply:
Showing two lines of headline sounds good, however, I'd have to check with Ben L what his reasonings were. // And here I go off on a tangent: Looking at dotcom, I'm still trying to get my head around what exactly is supposed to happen at the top of a picture essay. How deep is the picture supposed to be on compact devices and tablet portrait? On dotcom, it's currently not as deep as on immersives (not full screen), more like square. Is that right?

Reply:
I see what you mean @benwuersching on dotcom. It looks like dotcom-rendering is more fullscreen and shows the first two lines of the headline https://www.theguardian.com/artanddesign/2020/may/21/mind-the-gap-two-metres-apart-masked-ridley-road-hackney-a-photo-essay?dcr

Seen as that's a newer design I assume that's the way they are going?

Reply:
@webb04 Hello again (was away yesterday). Yes, picture essay should look as feature-y as possible. But I still have to better understand the thinking behind dotcom-rendering. For picture essays, dotcom goes "more fullscreen" as you write, but not quite as fullscreen as immersive. Here is a desktop comparison (it's like they show one headline line less).
![picture_essay_desktop](https://user-images.githubusercontent.com/23035818/83009889-5937bc80-a00f-11ea-92b5-3667fbc86777.png)
![immersive_desktop](https://user-images.githubusercontent.com/23035818/83010010-84221080-a00f-11ea-8add-b90b6eba4c85.png)